src/d1to2fix/main.d

```diff
@@ -102,14 +102,32 @@ int main ( string[] args )
         return 1;
     }
 
+    // Get file name list to convert
+
+    import std.algorithm : splitter, filter;
+    import std.array;
+    import std.file : readText;
+
+    string[] fileNames;
+
+    if (args[1][0] == '@')
+    {
+        fileNames = readText(args[1][1 .. $])
+            .splitter("\n")
+            .filter!(s => s.length)
+            .array();
+    }
+    else
+        fileNames = args[1 .. $];
+
```

Comment:
Mmm, so is either `d1to2fix @file` or `d1to2fix file1 ... fileN` but `d1to2fix @files1 file1` can't be used? In that case (or in any case) why not using a command line option like -f (or even stdin) to get multiple files? I always found `@file` super weird...